src/atc/configurator/configurator.py

```diff
@@ -220,6 +223,30 @@ def _get_item_property(
         # exception for any missing key, which will give a meaningful error to the user.
         return raw_string.format(**replacements)
 
+    def _get_schema_property(
```

Comment:
I think you can remove this whole method. Don't you get an import loop when the Configurator in turn depends on the schema manager?